cmake/multi_image.cmake

```diff
@@ -27,6 +27,11 @@ if(IMAGE_NAME)
   share("set(${IMAGE_NAME}KERNEL_ELF_NAME ${KERNEL_ELF_NAME})")
   share("list(APPEND ${IMAGE_NAME}BUILD_BYPRODUCTS ${PROJECT_BINARY_DIR}/${KERNEL_HEX_NAME})")
   share("list(APPEND ${IMAGE_NAME}BUILD_BYPRODUCTS ${PROJECT_BINARY_DIR}/${KERNEL_ELF_NAME})")
+
+  if("${IMAGE_NAME}" STREQUAL  "spm_")
+	  share("set(${IMAGE_NAME}CONFIG_FPU ${CONFIG_FPU})")
```

Comment:
I think this should go here instead:
https://github.com/nrfconnect/sdk-nrf/blob/11fb11eb1cbda80e9a83806e5f5499f8ce6a2cc6/subsys/spm/CMakeLists.txt#L16

Will also allow you to remove the `if .. endif`.